datasource_api_client/models/datasource_dto.py

```diff
@@ -29,23 +33,23 @@ class DatasourceDto:
         owner_info (DatasourceOwnerInfo):
         project_ids (List[str]):
         status (DatasourceDtoStatus):
-        description (Union[Unset, None, str]):
+        description (Union[None, Unset, str]):
     """
 
-    added_by: DatasourceDtoAddedBy
+    added_by: "DatasourceDtoAddedBy"
     auth_type: DatasourceDtoAuthType
-    config: DatasourceConfig
+    config: "DatasourceConfig"
```

Comment:
Why is there a mix of Strings and variables for the type definitions?

Reply:
Very good point. I haven't found out why it's doing so. But seeing it's widely used in open api python generator and generated code. https://github.com/openapi-generators/openapi-python-client/blob/main/openapi_python_client/parser/openapi.py#L192